spa/src/components/pageEditor/editInterface/pageSetup/PublishWidget.styled.js

```diff
@@ -38,3 +38,17 @@ export const Or = styled.p`
   margin: 0;
   padding: 1rem;
 `;
+
+export const InvalidPaymentProvider = styled.div`
+  width: 100%;
+  color: #ff476c;
+  text-align: center;
+  padding: 12px 0px;
+  border-bottom: 1px solid;
+  border-color: #eee;
+
+  p {
+    color: #080708;
+    font-size: 90%;
```

Comment:
Not sure having `%` in font-size is good practice. We can use the `em` keyword that takes into account the parent's font-size or you could use `rem` that taks into account the html default size for the element (in this case default size for `p`)
```suggestion
    font-size: 0.9em;
```